### PR TITLE
changed alerting to two hours

### PR DESCRIPTION
**Pull request by wdanfort**

### CODE REVIEW

File: app/airq/models/clients.py [hunk 1]
```diff
@@ -146,12 +146,13 @@ class Client(db.Model):  # type: ignore
         ),
     )
 
-    # Send alerts at most every one hour to avoid spamming people.
+    # Send alerts at most every TWO hours to avoid spamming people.
```
Comment (ianhoffman) [inline app/airq/models/clients.py:149]: LGTM. Can you also update the README to reflect this, in the features section?

Comment (wdanfort) [reply app/airq/models/clients.py:149]: yeah fixed in seperate PR

Comment (ianhoffman) [reply app/airq/models/clients.py:149]: got it. in the future you can just update this one if that's easier:

* make your change
* `git add -A`
* `git commit <message>`
* `git push`

Comment (wdanfort) [reply app/airq/models/clients.py:149]: fixed!

**Review by ianhoffman — APPROVED** [review]

**Pull request merged**